src/cronjobs.js

```diff
@@ -0,0 +1,88 @@
+const fs = require('fs');
+const mustache = require('mustache');
+const templates = require('./templates');
+const { getNewestBlogEntry, getNewestTweet, getNewestVideo } = require('./mathworks-docs');
+
+const cronjob_data_file = './storage/cronjob_data.json';
+
+const cronjobs = [
+    {
+        name: 'Blog',
+        use: getNewestBlogEntry,
+        interval: 3 * 3600 * 1e3,
+        template: 'blog.md',
+        errors: [],
+    }, {
+        name: 'Twitter',
+        use: getNewestTweet,
+        interval: 1 * 3600 * 1e3,
+        template: 'twitter.md',
+        errors: [],
+    }, {
+        name: 'Youtube',
+        use: getNewestVideo,
+        interval: 2 * 3600 * 1e3,
+        template: 'youtube.md',
+        errors: [],
+    },
+];
+
+const cronjob_data = JSON.parse(fs.readFileSync(cronjob_data_file, 'utf8'));
```

Comment:
JSON files can be required directly without reading the filesystem fyi

Reply:
Good point, thanks!